Documentation/op-guide/v2-migration.md

```diff
@@ -28,6 +28,12 @@ Sometimes an etcd cluster will possibly have v3 data which should not be overwri
 ETCDCTL_API=3 etcdctl get "" --from-key --keys-only --limit 1 | wc -l
 ```
 
+In case v2 data was migrated with TTLs, ensure that stores are consistent by checking the hashes:
+
+```sh
+ETCDCTL_API=3 etcdctl endpoint hash --cluster
```

Comment:
Can this be `hashkv`? The naming should be consistent with the underlying RPC / client call if possible (e.g., what if etcdctl ever gets support for the old `hash` rpc)